src/Movie-Card/Movie-Card.scss

```diff
@@ -0,0 +1,77 @@
+@import '../styles.scss';
+
+.movie-card {
+    height: 90%;
+    width: auto;
+    padding: 0;
+    margin: 20px;
+    border-radius: 5px;
+    position: relative;
+    padding-left: 10px;
+    align-content: center;
+    font-size: 0.8em;
+    font-family: sans-serif;
+    color: whitesmoke;
+    font-kerning: normal;
+    text-shadow: 0ch;
+    outline-color: black;
+         &:hover {
+            cursor: pointer;
+            .hidden {
+                opacity: 1;
+                margin-left: 10px;
+            }
+            .background {
```

Comment:
this is how I'm making the background image opaque while keeping the text normal

Reply:
It works great! I always struggle with opacity and CSS